ethpm_cli/main.py

```diff
@@ -94,7 +94,9 @@ def parse_arguments():
     return parser
 
 
-def main(parser, logger):
+def main():
+    logger = get_logger()
```

Comment:
*nitpick*  maybe this functin would be more intuitively named `setup_cli_logger` or `setup_stdout_logging`.  I ended up having to go look for the implementation to see what this did.